### DISCUSSION TITLE
Unexpected MariaDB errors when .... ?

### DISCUSSION THREAD

**Discussion by centralhardware** [photoprism/photoprism#2541] [Ideas]
it will be cool if the mariadb error log is clean.

now it almost entirely consists of lines like: 

`2022-07-22 15:40:57 photoprism[photoprism] @  [172.18.0.3] ERROR 1062: Duplicate entry '331' for key 'PRIMARY' : INSERT INTO `details` (`photo_id`,`keywords`,`keywords_src`,`notes`,`notes_src`,`subject`,`subject_src`,`artist`,`artist_src`,`copyright`,`copyright_src`,`license`,`license_src`,`software`,`software_src`,`created_at`,`updated_at`) VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)`

**Comment by lastzero** [comment]
Of course it would be cool. Only problem is that I don't see this, so a bit of help to reproduce it would be much appreciated!

**Reply by lastzero** [reply]
As a first step, I recommend you pull our latest preview build and provide feedback on the improvements. Maybe the SQL error is already fixed with that. If you still want me to look at the issue, please give me the information I need to reproduce it. It seems like you ran cleanup and there are primary files missing (JPEGs)? Is there anything else that would be good to know? Thanks a lot!

**Reply by centralhardware** [reply]
It's not error in common sense, just insert can not insert data because constrain violation so database raise exception about it. to show the database that this is the expected behavior must be specified `ON CONFLICT DO NOTHING`. it's be helpful if mariadb sql error log used for checking sql errors.
